tests/pyccel/test_pyccel.py

```diff
@@ -82,26 +83,56 @@ def teardown(path_dir = None):
         elif not f.endswith(".py"):
             os.remove(file_name)
 
+def compare_pyth_fort_output_by_type( p_output, f_output, dtype=float ):
+
+    if dtype is str:
+        assert(p_output.strip()==f_output.strip())
+    elif dtype is complex:
+        p_output, f_output = compare_pyth_fort_output_by_type( p_output, f_output, float)
+        p_output, f_output = compare_pyth_fort_output_by_type( p_output, f_output, float)
```

Comment:
Can you explain why you call the function twice with the same arguments? Do the two calls process the real and imaginary parts, respectively?

Reply:
Yes exactly. I process 2 floats so the real then imaginary parts of a complex